### PR TITLE
Fix displaying media from URL

### PR DESCRIPTION
**Pull request by SlavyMihov**
Identify source and display accordingly media from URL or local storage.

### CODE REVIEW

File: DigitalSignageUAP/SlideshowPage.xaml.cs [hunk 1]
```diff
@@ -307,8 +307,20 @@ void PlayAudio()
                             DisplayObject DO = new DisplayObject();
                             string filename = fileElement.Attribute("path").Value.Substring(fileElement.Attribute("path").Value.LastIndexOf('/') + 1);
```
Comment (alecont) [inline DigitalSignageUAP/SlideshowPage.xaml.cs:308]: why don't we use the absolute path here? if the file is in a subfolder in the storage file location, this code will pick up the wrong file...

File: DigitalSignageUAP/SlideshowPage.xaml.cs [hunk 2]
```diff
@@ -307,8 +307,20 @@ void PlayAudio()
...
                             StorageFile file = await tmp.CreateFileAsync(filename, CreationCollisionOption.ReplaceExisting);
-                            byte[] bytes = File.ReadAllBytes(filename);
-                            await FileIO.WriteBufferAsync(file, WindowsRuntimeBufferExtensions.AsBuffer(bytes));
+                            Uri fileElementUri;
+
+                            if (Uri.TryCreate(fileElement.Attribute("path").Value, UriKind.Absolute, out fileElementUri)
```
Comment (alecont) [inline DigitalSignageUAP/SlideshowPage.xaml.cs:312]: looks good, but I have one question: I thought there was a regression somehow and reading files from a web location was broken... but from the PR it looks it never worked... is that correct?

File: DigitalSignageUAP/SlideshowPage.xaml.cs (hunk 1 above)
Comment (SlavyMihov) [inline DigitalSignageUAP/SlideshowPage.xaml.cs:308]: The local paths use backslashes ("\") while the URLs use forward slashes ("/"). Therefore LastIndexOf('/') above will only affect the URL paths. Filenames in subfolders on the local filesystem should be resolved correctly as defined in the Assets\config.xml file.

File: DigitalSignageUAP/SlideshowPage.xaml.cs (hunk 2 above)
Comment (alecont) [inline DigitalSignageUAP/SlideshowPage.xaml.cs:312]: chatted with Slavi, it never worked. works now :-)

**Pull request merged**